components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -9043,6 +9043,11 @@ def renderJpegRegion(self, z, t, x, y, width, height, level=None,
                     return self.renderJpeg(z, t, None)
             rv = self._re.renderCompressed(self._pd, self._conn.SERVICE_OPTS)
             return rv
+        except omero.ApiUsageException:
+            logger.debug('On renderJpegRegion')
+            logger.debug('Cannot set resolution %s' % level)
```

Comment:
This does not necessarily indicate that the resolution cannot be set as the `ApiUsageException` could have been thrown by any of the remote invocations here. I would not include this log line. You can also use the following form to avoid having to call the `traceback` module and end up logging separately:

```
...
logger.debug('On renderJpegRegion', exc_info=True)
...
```